Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```
@@ -0,0 +1,24 @@
+- type: entity
+  name: RB-4
+  parent: BaseWeaponLauncher
+  id: LPPWeaponCyborgLauncherRocket
+  description: Block of combat rockets for syndicate cyborg.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)